migrations/20200824194331-patient-gender.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.sequelize.transaction((t) => {
+      return Promise.all([
+        queryInterface.bulkUpdate(
+          'patients',
+          { gender: 'M' },
+          { gender: 'Male' },
+          { transaction: t }
+        ),
+        queryInterface.bulkUpdate(
+          'patients',
+          { gender: 'F' },
+          { gender: 'Female' },
+          { transaction: t }
+        ),
+        queryInterface.changeColumn(
+          'patients',
+          'gender',
+          {
+            type: Sequelize.DataTypes.ENUM('M', 'F'),
+          },
+          {
+            transaction: t,
+          }
+        ),
+      ]);
+    });
+  },
+
+  down: (queryInterface, Sequelize) => {
```

Comment:
same issues from the up migration need to be addressed here - note that non-male/female values can't be recovered so you don't have to worry about that here